web-components/src/[sandbox]/examples/combobox.ts

```diff
@@ -115,18 +119,14 @@ export const comboBoxTemplate = html`
     </div>
   </md-combobox>
   <h3>Slot Complex Object Content Multi</h3>
```

Comment:
Should this header since the combobox isn't is-multi anymore and is more about with-custom-content?

Reply:
Thanks! It's my error. I checked, and forgot return it property. Fixed